lib/rubocop/ast/node.rb

```diff
@@ -356,6 +368,11 @@ def empty_source?
         {assignment? (send _recv :<< ...)}
       PATTERN
 
+      def enumerating?(method)
+        (enumerating_assigned?(method) ||
```

Comment:
The other parens are redundant. I'm also thinking this would fit on a single line.